make: runner: Command not found

Kokoro is currently failing with the following;
```
make: Target 'tests' not remade because of errors.
Script done, file is /tmpfs/tmp/tmp.tE9C3ozVkR
+ TESTS_RET=2
+ set +x -e
----------------------------------------
A failure occurred during test running.
----------------------------------------
runner Slang --test chapter-26/26.2--package-decl.sv --out ./out//logs/Slang/chapter-26/26.2--package-decl.sv.log --quiet
make: runner: Command not found
Makefile:121: recipe for target 'out//logs/Slang/chapter-26/26.2--package-decl.sv.log' failed
make: [out//logs/Slang/chapter-26/26.2--package-decl.sv.log] Error 127 (ignored)
```

--- a/.github/kokoro/run.sh
+++ b/.github/kokoro/run.sh
@@ -23,7 +23,6 @@ echo "----------------------------------------"
 	conda info -a
 	conda env create --file conf/environment.yml
 	conda activate sv-test-env
-	pip install -r conf/requirements.txt
 )
 echo "----------------------------------------"
 

--- a/.github/travis/install.sh
+++ b/.github/travis/install.sh
@@ -17,3 +17,6 @@ conda update -q conda
 conda info -a
 conda env create --file conf/environment.yml
 conda activate sv-test-env
+
+set +e
+set +x

--- a/.github/travis/install.sh
+++ b/.github/travis/install.sh
@@ -1,0 +1,19 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# Add local submodules
+.github/add-local-submodules.sh "${TRAVIS_PULL_REQUEST_SLUG:-$TRAVIS_REPO_SLUG}"
+
+# Get a conda environment
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+bash miniconda.sh -b -p $HOME/miniconda
+source "$HOME/miniconda/etc/profile.d/conda.sh"
+hash -r
+conda config --set always_yes yes --set changeps1 no
+conda install -q setuptools
+conda update -q conda
+conda info -a
+conda env create --file conf/environment.yml
+conda activate sv-test-env

--- a/.travis.yml
+++ b/.travis.yml
@@ -26,18 +26,7 @@ jobs:
 
     - name: "Generate report.hml"
       install:
-        - .github/add-local-submodules.sh "${TRAVIS_PULL_REQUEST_SLUG:-$TRAVIS_REPO_SLUG}"
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - source "$HOME/miniconda/etc/profile.d/conda.sh"
-        - hash -r
-        - conda config --set always_yes yes --set changeps1 no
-        - conda install -q setuptools
-        - conda update -q conda
-        - conda info -a
-        - conda env create --file conf/environment.yml
-        - conda activate sv-test-env
-        - pip install -r conf/requirements.txt
+        - source ./.github/travis/install.sh
       script:
         - "make generate-tests"
         - "make tests USE_ALL_RUNNERS=1 -j2"

--- a/Makefile
+++ b/Makefile
@@ -34,16 +34,19 @@ runners:
 # $(1) - runner name
 # $(2) - test
 define runner_test_gen
+
+# Set the runner value for the log target
 ifneq ($(USE_CGROUP),)
-RUNNER := cgexec -g memory,cpu:$(USE_CGROUP)/$(1) ./tools/runner
+$(OUT_DIR)/logs/$(1)/$(2).log : RUNNER = cgexec -g memory,cpu:$(USE_CGROUP)/$(1) ./tools/runner
 else
-RUNNER := ./tools/runner
+$(OUT_DIR)/logs/$(1)/$(2).log : RUNNER = ./tools/runner
 endif
 
 $(OUT_DIR)/logs/$(1)/$(2).log: $(TESTS_DIR)/$(2) | $(1)-cg
-	$(RUNNER) --runner $(1) --test $(2) --out $(OUT_DIR)/logs/$(1)/$(2).log --quiet
+	$$(RUNNER) --runner $(1) --test $(2) --out $(OUT_DIR)/logs/$(1)/$(2).log --quiet
 
 tests: $(OUT_DIR)/logs/$(1)/$(2).log
+
 endef
 
 # $(1) - runner name

--- a/conf/environment.yml
+++ b/conf/environment.yml
@@ -14,3 +14,6 @@ dependencies:
   - verilator
   - yosys
   - zachjs-sv2v
+  - pip
+  - pip:                            # Packages installed from PyPI
+    - -r file:requirements.txt

--- a/tools/runner
+++ b/tools/runner
@@ -65,6 +65,8 @@ os.environ['PATH'] = ":".join(new_path)
 runner = os.path.abspath(os.path.join(dirs['runners'], args.runner))
 out = os.path.abspath(args.out)
 
+os.makedirs(os.path.dirname(out), exist_ok=True)
+
 if args.version:
     version = runner_obj.get_version()
     with open(out, "w") as f:

